Tracing Web Service client: {urn:hello:wsdl}HelloService
Outbound SOAP message:
<S:Envelope xmlns:S="http://schemas.xmlsoap.org/soap/envelope/">
  <S:Body>
    <sayHello xmlns="urn:hello:wsdl">
      <name>friend</name>
    </sayHello>
  </S:Body>
</S:Envelope>
Tracing Web Service client: {urn:hello:wsdl}HelloService
Outbound SOAP message:
<S:Envelope xmlns:S="http://schemas.xmlsoap.org/soap/envelope/">
  <S:Header/>
  <S:Body>
    <CipherBody>ZbeYI5k7yR89oyIEnBSZD+6Up02wRMrKgq2P++B/0qkhcpinJzNyowqFeROdxLpqPU5hUkbeEXJ5&#13;
XPv9+v8ux+v7ne18QTX+E01jamk0rkjeWjSghLa6JVGIzGGtcXDHOlLu8zizPGf3QxIIiqvW14ET&#13;
ewnwAr9DaiILTgoJbFA7Y6a5849yAnxZY1MqkUy7wSpeOA1KFQthz/7QKvMjIjfibR29okKSS2vX&#13;
XU0tJRw=&#13;
</CipherBody>
  </S:Body>
</S:Envelope>
Tracing Web Service client: {urn:hello:wsdl}HelloService
Inbound SOAP message containing a Fault:
<S:Envelope xmlns:S="http://schemas.xmlsoap.org/soap/envelope/">
  <S:Header/>
  <S:Body>
    <S:Fault xmlns:ns3="http://www.w3.org/2003/05/soap-envelope">
      <faultcode>S:Server</faultcode>
      <faultstring>Test Exception generated in method interceptMessage-server-outbound</faultstring>
    </S:Fault>
  </S:Body>
</S:Envelope>
Tracing Web Service client: {urn:hello:wsdl}HelloService
Inbound SOAP message containing a Fault forced back to client:
<S:Envelope xmlns:S="http://schemas.xmlsoap.org/soap/envelope/">
  <S:Header/>
  <S:Body>
    <S:Fault>
      <faultcode>S:Client</faultcode>
      <faultstring>Extension failed to decipher message</faultstring>
    </S:Fault>
  </S:Body>
</S:Envelope>
